addons/main/script_macros.hpp

```diff
@@ -1,11 +1,4 @@
-// BWC for CBA's DEBUG_SYNCHRONOUS - https://github.com/CBATeam/CBA_A3/pull/466/
-#ifdef CBA_DEBUG_SYNCHRONOUS
-    // For New CBA:
-    #define DEBUG_SYNCHRONOUS
-    // For Old CBA:
-    #define CBA_fnc_log { params ["_file","_lineNum","_message"]; diag_log [diag_frameNo, diag_tickTime, time,  _file + ":"+str(_lineNum + 1), _message]; }
-#endif
-
+#define DEBUG_SYNCHRONOUS
```

Comment:
@PabstMirror ^ debug synchronous is always enabled in ACE. I don't think it makes sense to ever have that off.